templates/localgov_alert_banner.html.twig

```diff
@@ -9,14 +9,47 @@
  * Available variables:
  * - content: A list of content items. Use 'content' to print all content, or
  * - attributes: HTML attributes for the container element.
+ * - display_title: Boolean indicating if alert banner title should be displayed
+ * - remove_hide_link: Boolean indicating if the link to close the banner should be hidden
+ * - type_of_alert: Type of alert banner
  *
  * @see template_preprocess_localgov_alert_banner()
  *
  * @ingroup themeable
  */
 #}
-<div{{ attributes.addClass('localgov_alert_banner') }}>
-  {% if content %}
-    {{- content -}}
-  {% endif %}
+{{ attach_library('localgov_alert_banner/alert_banner') }}
+{% set has_link = content.link is not empty %}
+{% set type_of_alert =  type_of_alert|split('--')[1] %}
+{% set classes = [
+    'wrapper',
+    'js-alert-banner',
+    'localgov_alert_banner--' ~ type_of_alert,
+    is_front ? 'localgov_alert_banner--homepage' : '',
+    has_link ? 'localgov_alert_banner--has-link' : 'localgov_alert_banner--no-link'
+  ]
+%}
+<!-- Alert Banner-->
+<div class="localgov_alert_banner" role="alert">
+  <div{{ attributes.addClass(classes) }}>
+    <div class="localgov_alert_banner--inner">
+      <div class="localgov_alert_banner--inner--content">
+        {% if display_title %}
+          <h2 class="localgov_alert_banner--content-heading">{{ content.title }}</h2>
+        {% endif %}
+        <p class="localgov_alert_banner--content-body">{{ content.short_description }}</p>
+        {% if content.link %}
+          <div class="localgov_alert_banner--content-link">{{ content.link }}</div>
+        {% endif %}
+      </div>
+      <div class="localgov_alert_banner--actions paragraph">
+        {% if not remove_hide_link %}
```

Comment:
Ideally move the if outside the wrapper div block so it only renders if the hide link needs to be.
We can also remove the paragraph class as that comes from BHCC theme.

Reply:
Thanks @andybroomfield that all makes sense, I'll get on with the changes.